eng/helix/content/RunTests/TestRunner.cs

```diff
@@ -213,50 +213,60 @@ public async Task<int> RunTestsAsync()
             try
             {
                 // Timeout test run 5 minutes before the Helix job would timeout
-                var cts = new CancellationTokenSource(Options.Timeout.Subtract(TimeSpan.FromMinutes(5)));
+                var testProcessTimeout = Options.Timeout.Subtract(TimeSpan.FromMinutes(5));
+                var cts = new CancellationTokenSource(testProcessTimeout);
                 var diagLog = Path.Combine(Environment.GetEnvironmentVariable("HELIX_WORKITEM_UPLOAD_ROOT"), "vstest.log");
-                var commonTestArgs = $"test {Options.Target} --diag:{diagLog} --logger:xunit --logger:\"console;verbosity=normal\" --blame \"CollectHangDump;TestTimeout=15m\"";
+                var commonTestArgs = $"test {Options.Target} --diag:{diagLog} --logger:xunit --logger:\"console;verbosity=normal\" --blame \"CollectHangDump;CollectDump;TestTimeout=15m\"";
```

Comment:
Do we want this to use the same (new syntax) arguments as 7.0?

Reply:
May as well 👍